src/dotnet/Fable.Compiler/Transforms/Transforms.Util.fs

```diff
@@ -122,6 +122,8 @@ module AST =
     let makeNumConst (x: float) = NumberConstant (float x, Float64) |> Value
     let makeDecConst (x: decimal) = NumberConstant (float x, Float64) |> Value
 
+    let argTypes (args: Expr list) = args |> List.map (fun a -> a.Type)
```

Comment:
This is actually not called, right? Maybe we should remove it to avoid confusion with `info.ArgTypes` as I explain in the comment to the PR.